explorer/src/components/Farming/index.tsx

```diff
@@ -0,0 +1,240 @@
+            <ol className='list-inside list-decimal text-left'>
+              <li>Download the tar.gz file from the link above.</li>
+              <li>Extract the file to your desired location.</li>
+              <li>Open a terminal and navigate to the extracted folder.</li>
+              <li>Run `./install.sh` to install Space Acres.</li>
+            </ol>
+          </div>
+        )
+      default:
+        return <p className='text-center'>Sorry, your operating system is not supported.</p>
+    }
+  }, [getDownloadLink, userOS])
+
+  const downloadButton = useMemo(
+    () => (
+      <div className='mb-2 flex items-center justify-center text-center'>
+        <a href={getDownloadLink(userOS || '')} className='row btn-download'>
+          <button className='relative mb-2 w-full cursor-pointer rounded-full bg-purpleAccent from-pinkAccent to-purpleDeepAccent py-[10px] pl-3 pr-16 text-left font-["Montserrat"] text-white shadow-md focus:outline-none focus-visible:border-indigo-500 focus-visible:ring-2 focus-visible:ring-white focus-visible:ring-opacity-75 focus-visible:ring-offset-2 focus-visible:ring-offset-orange-300 dark:bg-gradient-to-r dark:text-white sm:text-sm md:pr-10'>
+            Download Space Acres
+          </button>
+          <div className='text-sm text-gray-900 dark:text-white'>
+            {getAssetName(userOS || '')} for {userOS}
+          </div>
+        </a>
+      </div>
+    ),
+    [getDownloadLink, getAssetName, userOS],
+  )
+
+  return (
+    <div className='flex w-full flex-col items-center space-y-4'>
+      <div className='w-full max-w-4xl'>
+        <div className='mb-4 w-full rounded-[20px] border border-slate-100 bg-white px-3 py-4 shadow dark:border-none dark:bg-gradient-to-r dark:from-gradientTwilight dark:via-gradientDusk dark:to-gradientSunset sm:p-6'>
+          <div className='mb-10 flex flex-col items-center justify-center'>
+            <h1 className='mb-8 mt-6 text-center text-4xl font-bold text-gray-900 dark:text-white'>
+              Put your unused disk space to work and contribute to the Network
+            </h1>
+            {renderDownloadSection && downloadButton}
+          </div>
+          <div className='m-6 flow-root'>
+            <Image
+              alt='Space Acres Screenshot Installation'
+              width={800}
+              height={400}
+              src='https://docs.subspace.network/assets/images/space-acres-setup-7-3490cba2e75635efdea0006d06da9936.png'
+              className='mx-auto'
+            />
+          </div>
+        </div>
+
+        <div className='m-4 p-4'>
+          <h3 className='mb-8 mt-6 text-center text-4xl font-bold text-gray-900 dark:text-white'>
+            By contributing storage and compute to the network, you play a crucial role in securing
+            it, while also earning rewards.
+          </h3>
+          <h3 className='mb-8 mt-6 text-center text-4xl font-bold text-gray-900 dark:text-white'>
```

Comment:
```suggestion
         
```

Reply:
I had a second thought that we shouldn't put this wording in the first version :)